stage1/init/pod.go

```diff
@@ -506,7 +511,24 @@ func (p *Pod) appToNspawnArgs(ra *schema.RuntimeApp) ([]string, error) {
 			opt[0] = "--bind="
 		}
 
-		opt[1] = vol.Source
+		switch vol.Kind {
+		case "host":
+			opt[1] = vol.Source
+		case "empty":
+			var absRoot string
+			var err error
+			if p.Root == "." {
```

Comment:
[filepath.Abs](https://golang.org/pkg/path/filepath/#Abs) can be used instead of manually getting the working directory.